### DISCUSSION TITLE
Is there a way to trigger "three fourths" of the display?

### DISCUSSION THREAD

**Discussion by thornomad** [rxhanson/Rectangle#306] [Q&A]
Hello! Just switched over from Spectacle thanks for your contributions on this everyone!

As the title suggests: is there a way to do three fourths (3/4ths) of the display?  I would like to put one window at 3/4 and another at 1/4.  I read through the custom terminal settings but am not imagining a way I could modify this to work (yet).

Any advice appreciated!

**Comment by rxhanson** [comment]
If you execute 2nd 4th or 3rd 4th twice, the second execution is 3/4.

**Reply by thornomad** [reply]
Thank you so much for that quick reply!  I hadn't tried that!  This seems to perform correctly on a landscape monitor; however, on my portrait monitor it behaves differently: cycling between a one fourth sized window and a one half sized window.  Here is a video to demonstrate.  

https://user-images.githubusercontent.com/124717/102998138-96582480-44f4-11eb-9f88-2d71c83b5bb9.mov

Maybe this is an issue only affecting portrait layouts? I am not familiar with Swift or I would take a look myself.  Thanks again for your quick response.
